### ISSUE TITLE
bm_subtract does not behave as expected in 2024.4

### ISSUE THREAD

**Issue by KormexGit** [YoYoGames/GameMaker-Bugs#5367]
### Description
Using bm_subtract behaves very differently in beta compared to monthly.
Here's what the sample project looks like in 2024.2:

![image](https://github.com/YoYoGames/GameMaker-Bugs/assets/100942509/f88ec372-394d-4a0e-8b2b-9f49c4fa0570)

And here's what it looks like in 2024.4:

![image](https://github.com/YoYoGames/GameMaker-Bugs/assets/100942509/4b7e94f4-cf7e-496f-9bb3-0e8b9579222b)

### Steps To Reproduce
```
if !surface_exists(surf) {
	surf = surface_create(display_get_gui_width(), display_get_gui_height());
}

surface_set_target(surf);

draw_clear_alpha(c_black, 1);

gpu_set_blendmode(bm_subtract);
draw_sprite_ext(Sprite1, 0, x, y, 1, 1, 0, c_white, 1);
gpu_set_blendmode(bm_normal);

surface_reset_target();

draw_surface(surf, 0, 0);
```

### Which version of GameMaker are you reporting this issue for?
IDE v2024.400.0.532 Runtime v2024.400.0.551

### Which operating system(s) are you seeing the problem on?
Windows 10.0.22631.0

[0ef30a62-eaae-443e-97fa-650b634f18a6](https://api.gamemaker.io/api/github/downloads/0ef30a62-eaae-443e-97fa-650b634f18a6)

**Comment by L-Clouds** [comment]
Have to imagine this is connected to #4859

**Comment by JonnoRobson** [comment]
This should now be fixed as of the latest BETA (IDE v2024.400.0.664 Runtime v2024.400.0.762).
